script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -46,94 +46,94 @@ The man concentrates the cold waves flooding out of himself into a single point,
 <Page11>
 "...It's a shame, but die.
 Don't go and make my heart pound like that."
-The chunks of ice get expelled like a pulley sliding through the frozen atmosphere...!
+The chunks of ice get expelled like a pulley sliding through the frozen atmosphere...!//Wat 凍った大気を滑る滑車のように、氷塊を撃ち出してくる……！
```

Comment:
should be are rather than get

Reply:
Do you have any idea what this pulley stuff is supposed to mean? Smooth movement? Acceleration? Maybe Nasu just likes kinematics and wanted to drop some simple machine references?